testing/src/main/kotlin/org/neo4j/connectors/kafka/testing/sink/Neo4jSinkRegistration.kt

```diff
@@ -113,6 +117,27 @@ internal class Neo4jSinkRegistration(
     RegistrationSupport.unregisterConnector(URI("$connectBaseUri/connectors/$name/"))
   }
 
+  fun getTaskState(): String {
+    val request =
+        HttpRequest.newBuilder(URI("$connectBaseUri/connectors/$name/status"))
+            .header("Content-Type", "application/json")
+            .header("Accept", "application/json")
+            .GET()
+            .build()
+
+    val response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString())
+    if (response.statusCode() != 200) {
+      throw RuntimeException(
+          "Could not get connector status, response code: ${response.statusCode()}")
+    }
+
+    // by default task count is 1
```

Comment:
maybe add an assertion to check that we have only 1 task?

Reply:
yes, sounds good 👍